atc/config.go

```diff
@@ -514,6 +514,9 @@ func DefaultTLSConfig() *tls.Config {
 
 		// Security team recommends a very restricted set of cipher suites
 		CipherSuites: []uint16{
+			tls.TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,
+			tls.TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384,
+			tls.TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305,
```

Comment:
Is this intentionally added or is this a merge artifact?

Reply:
We had some issues with TLS handshakes with the suites configured so had to, somewhat reluctantly, add the most secure cipher suites [autocert uses](https://github.com/golang/crypto/blob/master/acme/autocert/autocert.go#L356). We put this through SSL Labs to verify the suites we added are in fact secure.

I'm open to alternative ideas.

Reply:
Sounds good, just making sure it wasn't a conflict as we happened to touch that code very recently:  https://github.com/concourse/concourse/commit/6f8880a7dcb87a0bee102820855959f7fd4b30f7